components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/SubscriptionValidationDAO.java

```diff
@@ -358,19 +358,17 @@ public GlobalPolicy getGlobalPolicyByNameForTenant(String policyName, String ten
      * @param tenantId : unique identifier of tenant
      * @return {@link List<API>}
      * */
-    public List<API> getAllApis(String organization) {
+    public List<API> getAllApis(String organization, boolean isExpand) {
 
-        String sql = SubscriptionValidationSQLConstants.GET_ALL_APIS_BY_ORGANIZATION;
+        String sql = SubscriptionValidationSQLConstants.GET_ALL_APIS_BY_ORGANIZATION_AND_DEPLOYMENT_SQL;
```

Comment:
Shall we remove the GET_ALL_APIS_BY_ORGANIZATION query from the list of SQLs as well.. seems like it is not used after this change..